src/index.js

```diff
@@ -48,6 +48,17 @@ function testUniqueness( element, selector )
 }
 
 /**
+ * Tests all selectors for uniqueness and returns the first unique selector.
+ * @param  { Object } element
+ * @param  { Array } selectors
+ * @return { String }
+ */
+function getFirstUnique( element, selectors )
+{
+    return selectors.find( testUniqueness.bind( null, element ) );
```

Comment:
No it's not because `testUniqueness()` expects the `element` as its first parameter which `.find()` will not give it. It will simply call the function with the `selector` as its first argument. Hence the `.bind(null, element)` which returns a function that can be called with the `selector` as its first argument and calls `testUniqueness(element, selector)`.